ebuild-writing/variables/text.xml

```diff
@@ -588,5 +588,54 @@ when joining paths. For example: <c>${D%/}/</c>, <c>${ED%/}/</c>,
 </body>
 </section>
 
+<section>
+<title>Use of constant-value variables in ebuilds</title>
+<body>
+
+<p>
+Variables frequently have significant value in ebuilds, making it
+possible to avoid unnecessary repetitions and make maintenance easier.
+However, references to constant-value variables should be used with care
+as their proactive use can harm readability and increase maintenance
+burden (e.g. when renaming the package). In particular, using variables
+whose values are not strictly guaranteed to match the expected string
+should be avoided.
+</p>
+
+<p>The examples of beneficial constant-value variable references are:</p>
+<ul>
+  <li>
+    using <c>${PV}</c> and the variables derived from it (<c>${P}</c>,
+    <c>${MY_P}</c>…) in <c>SRC_URI</c> and <c>S</c> to avoid having
+    to update those variables for every version bump (presuming that
+    <c>${PV}</c> is used to indicate the upstream version);
+  </li>
+
+  <li>
+    using <c>${PF}</c> in <c>--docdir</c> path — this is a canonical
+    Gentoo path that is always required to match <c>${PF}</c>.
```

Comment:
Those are two separate problems. Here I'm just telling it's fine to use. The formal requirement for using it belongs elsewhere.